build/circle-trigger-nightly.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -eu
+
+REV=$(git describe --tags)
```

Comment:
I thought about the best way to do it, but I didn't see a benefit of making this more complicated (the way I think about it, it's much like running `make test`, but on CI, so your currently checked out commit doesn't seem so surprising).
I don't see a reason to special-case `master` somehow (this should work for anything that's in our central repo), and I didn't find an extra argument to deal with/rejuggling the deploy section in our `circle.yaml` worth it. Might change, but I'll leave as is for now.